Open-Rival/include/TimerUtils.h

```diff
@@ -1,11 +1,30 @@
 #ifndef TIMER_UTILS_H
 #define TIMER_UTILS_H
 
+#include <chrono>
+
 namespace Rival {
     namespace TimerUtils {
 
         static const int timeStepMs = 1000 / 60;
 
+        using MillisecondsType = std::chrono::milliseconds;
+        using MillisecondsRep = decltype(MillisecondsType());
+        using TimeType = decltype(std::chrono::system_clock::now());
+
+        static const auto gameStartTime(
+            std::chrono::duration_cast<MillisecondsType>(
+                std::chrono::system_clock::now().time_since_epoch()));
```

Comment:
If it helps, we can use [`SDL_GetTicks`](https://wiki.libsdl.org/SDL_GetTicks).

Reply:
<chrono> is part of STL, and is guaranteed (by standard-compliant vendors) to do what it claims. SDL_* methods tries to implement non-standard functionalities, and may not be cross-platform compatible in each and every detail. Since we are using modern C++, we should get this done properly as long as it does not break SDL methods. In this case, this logic has nothing to do with SDL.

Reply:
Fair enough.